packages/dev/core/src/Shaders/ShadersInclude/pbrBlockSubSurface.fx

```diff
@@ -340,11 +340,13 @@ struct subSurfaceOutParams
         #endif
 
         #ifdef SS_DISPERSION
-        float realIOR = 1.0 / ior;
-            float iorDispersionSpread = 0.02 * dispersion * (realIOR - 1.0);
-            vec3 iors = vec3(ior + iorDispersionSpread, ior, ior - iorDispersionSpread);
+            float realIOR = 1.0 / ior;
+            float iorDispersionSpread = 0.025 * dispersion * (realIOR - 1.0);
+            vec3 iors = vec3(ior - iorDispersionSpread, ior, ior + iorDispersionSpread);

```

Comment:
Is it not `vec3 iors = vec3(1.0/(realIOR - iorDispersionSpread), ior, 1.0/(realIOR + iorDispersionSpread));` instead?

Reply:
Quite right. Done.